rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TclLauncherUi.xtend

```diff
@@ -112,14 +112,14 @@ class TclLauncherUi implements Launcher {
 	private def void safeUpdateJunitTestView(String elementId, File expectedFile) {
 		val parentFolder = new File(expectedFile.parent)
 		if (!parentFolder.exists && !parentFolder.mkdirs) {
-			logger.error('''failed to change into parentFolder='{}' ''', parentFolder.absolutePath)
+			logger.error("failed to change into parentFolder='{}'", parentFolder.absolutePath)
 		} else {
 			val newFile = File.createTempFile(expectedFile.name, ".xml", parentFolder)
 			if (expectedFile.exists) {
 				try {
 					Files.move(expectedFile, newFile)
 				} catch (IOException e) {
-					logger.error("error during storage of test run result " + expectedFile.path, e)
+					logger.error("error during storage of test run result {}", expectedFile.path, e)
```

Comment:
In order to pass the exception to the logger, only one additional parameter may be passed => appending is in case of errors ok (if passing an exception).
The change for the warnings are absolutely right. Thanks for the correction.